subprojects/android-test/snackbar-rule/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
Why do you decide to put it into a separate module? IMHO it's an overkill to make two modules for every rule or check in our test framework.

Reply:
I did it like for toast-rule. But we could overthink this strategy

Reply:
At least, It's good to separate classes that you can use without our test library. All further separations are questionable for me.

Reply:
Same, but looking from user's perspective i would be glad to have a choice not to apply features i don't need